### PR TITLE
Loans draw and cratio improvements

### PR DESCRIPTION
**Pull request by bachstatter**
- User safe cratio buffer rather than hardcoded safe c-ratio
 Fix Draw screen 
- Prevent user from adding more than max draw.
- Disable if you're below safe c-cratio
- Max button  now adds max debt to safe c-ratio (150%)

### CODE REVIEW

File: sections/loans/components/ActionBox/ActiveBorrowsTab/ModifyLoanActions/Draw.tsx [hunk 1]
```diff
@@ -62,6 +84,7 @@ const Repay: React.FC<RepayProps> = ({ loan, loanId, loanTypeIsETH, loanContract
 						onTxConfirmed: () => {},
 					}),
 			});
+			router.push('/loans/list');
 		} catch {
```
Comment (fritzschoff) [inline sections/loans/components/ActionBox/ActiveBorrowsTab/ModifyLoanActions/Draw.tsx:88]: guess you can remove the catch?

File: sections/loans/components/ActionBox/ActiveBorrowsTab/ModifyLoanActions/Draw.tsx [hunk 2]
```diff
@@ -62,6 +84,7 @@ const Repay: React.FC<RepayProps> = ({ loan, loanId, loanTypeIsETH, loanContract
 						onTxConfirmed: () => {},
 					}),
 			});
+			router.push('/loans/list');
```
Comment (fritzschoff) [inline sections/loans/components/ActionBox/ActiveBorrowsTab/ModifyLoanActions/Draw.tsx:87]: dunno if the routes.ts is still maintained?

**Review by fritzschoff — APPROVED** [review]

File: sections/loans/components/ActionBox/ActiveBorrowsTab/ModifyLoanActions/Draw.tsx (hunk 2 above)
Comment (bachstatter) [reply sections/loans/components/ActionBox/ActiveBorrowsTab/ModifyLoanActions/Draw.tsx:87]: Yea I think it is. Nice catch, should be using constants

File: sections/loans/components/ActionBox/ActiveBorrowsTab/ModifyLoanActions/Draw.tsx (hunk 1 above)
Comment (bachstatter) [reply sections/loans/components/ActionBox/ActiveBorrowsTab/ModifyLoanActions/Draw.tsx:88]: I take a look and try to handle errors properly

**Pull request merged**